roles/kubernetes-apps/external_provisioner/local_volume_provisioner/templates/local-volume-provisioner-cm.yml.j2

```diff
@@ -22,6 +22,7 @@ data:
     - {{ nodelabel }}
 {% endfor %}
 {% endif %}
+  useNodeNameOnly: "{{ local_volume_provisioner_use_node_name_only }}"
```

Comment:
With `local_volume_provisioner_use_node_name_only` as a boolean, add an if statement in order not to add useNodeNameOnly when it is false (default local-volume-provisioner behaviour and prevent current behaviour to stay the same)

Reply:
done